src/WebSite/Bit.Platform.WebSite/Web/Shared/Urls.cs

```diff
@@ -0,0 +1,33 @@
+﻿namespace Bit.Platform.WebSite.Web.Shared
+{
+    public static class Urls
+    {
+        public const string HomePage = "/";
+
+        public const string Projects = "/Projects";
+
+        public const string Github = "https://github.com/bitfoundation/bitplatform";
+
+        public const string GithubRepo = "https://github.com/bitfoundation/bitplatform";
+
+        public const string GithubIssues = "https://github.com/bitfoundation/bitplatform";
+
+        public const string GithubOrg = "https://github.com/bitfoundation/bitplatform";
+
+        public const string Linkedin = "https://www.linkedin.com/company/bit-foundation/about/";
+
+        public const string Tweter = "#";
```

Comment:
The string name should be `Twitter` and its link is [https://twitter.com/bitplatformhq](https://twitter.com/bitplatformhq)